nilearn/tests/test_ndimage.py

```diff
@@ -21,6 +25,48 @@ def test_largest_cc():
     b[5, 5, 5] = 1
     np.testing.assert_equal(a, largest_connected_component(b))
 
+    # Tests for correct errors, when an image or string are passed.
+    img = testing.generate_labeled_regions(shape = (10,11,12),
```

Comment:
PEP: Missing white space after ",". Should be like shape=(10, 11, 12) and n_regions=2)